frontend/public/style/_overrides.scss

```diff
@@ -96,6 +96,15 @@ kbd {
   max-width: 100%;
 }
 
+.pf-c-form {
```

Comment:
@sg00dwin, this is less than ideal, but couldn't figure out another fix.  Ideas?